src/course-unit/__mocks__/courseUnit.js

```diff
@@ -0,0 +1,83 @@
+module.exports = {
```

Comment:
We already have mock for unit data https://github.com/raccoongang/frontend-app-course-authoring/blob/Peter_Kulko/add-new-unit-btn/src/course-unit/__mocks__/courseUnitIndex.js or you need another one?

Reply:
This is a separate data mock for a new unit